classifier_and_summarizer/classifier/classifier.py

```diff
@@ -0,0 +1,30 @@
+class Classifier:
+    '''Class to determine if a webpage is stampable
+
+    Detemines if a webpage is stampable based on the following
+    max(media_count,text_count) + embedded_content_count >= min_pages
```

Comment:
Why max(media_count, text_count)?

Reply:
max is used here since even after some text/media is unmatched it will still be used solely as a stamp page content!

Reply:
Cool!